src/js/providers/html5.js

```diff
@@ -337,13 +337,15 @@ define([
 
         function _playingHandler() {
             _this.setState(states.PLAYING);
+            // For accuracy, trigger firstFrame before manipulating the DOM
+            _this.trigger(events.JWPLAYER_PROVIDER_FIRST_FRAME, {});
+
```

Comment:
I don't think we're loosing more than a millisecond adding attributes to the video tag. We try to trigger player events at the end of function (after any mutation) to avoid any errors caused by event handler changes.